### ISSUE TITLE
setup.py has tensorflow as dependency ( which can be tensorflow_gpu )

### ISSUE THREAD

**Issue by kumarsameer** [merantix/picasso#11]
* picasso version: 0.1.1
* Python version: 3.5.2
* Operating System: ubuntu -16.04

### Description
setup.py has dependency mentioned as tensorflow, which pulls in tensorflow package although my machine uses tensorflow_gpu package ( which is already updated ). 

### What I Did
sudo -H pip3 install picasso-viz

```
Paste the command(s) you ran and the output.
Collecting picasso-viz
  Using cached picasso_viz-0.1.1-py2.py3-none-any.whl
Collecting requests>=2.13.0 (from picasso-viz)
  Downloading requests-2.17.3-py2.py3-none-any.whl (87kB)
    100% |████████████████████████████████| 92kB 187kB/s 
Requirement already satisfied: Jinja2>=2.9.5 in /usr/local/lib/python3.5/dist-packages (from picasso-viz)
Collecting Flask>=0.12 (from picasso-viz)
  Downloading Flask-0.12.2-py2.py3-none-any.whl (83kB)
    100% |████████████████████████████████| 92kB 6.4MB/s 
Requirement already satisfied: protobuf>=3.2.0 in /usr/local/lib/python3.5/dist-packages (from picasso-viz)
Requirement already satisfied: six>=1.10.0 in /usr/lib/python3/dist-packages (from picasso-viz)
Requirement already satisfied: scipy>=0.18.1 in /usr/local/lib/python3.5/dist-packages (from picasso-viz)
Requirement already satisfied: MarkupSafe>=0.23 in /usr/local/lib/python3.5/dist-packages (from picasso-viz)
Requirement already satisfied: pyparsing>=2.1.10 in /usr/local/lib/python3.5/dist-packages (from picasso-viz)
Collecting Keras>=1.2.2 (from picasso-viz)
  Downloading Keras-2.0.4.tar.gz (199kB)
    100% |████████████████████████████████| 204kB 208kB/s 
Collecting olefile>=0.44 (from picasso-viz)
  Downloading olefile-0.44.zip (74kB)
    100% |████████████████████████████████| 81kB 6.3MB/s 
Requirement already satisfied: cycler>=0.10.0 in /usr/local/lib/python3.5/dist-packages (from picasso-viz)
Collecting Pillow>=4.0.0 (from picasso-viz)
  Downloading Pillow-4.1.1-cp35-cp35m-manylinux1_x86_64.whl (5.7MB)
    100% |████████████████████████████████| 5.7MB 107kB/s 
Collecting tensorflow>=1.0.0 (from picasso-viz)
  Downloading tensorflow-1.1.0-cp35-cp35m-manylinux1_x86_64.whl (31.0MB)
    3% |█▎                              | 1.2MB 260kB/s eta 0:01:55^C
Operation cancelled by user

``

**Comment by rhsimplex** [comment]
Hi @kumarsameer -- I'll look into a workaround.  Either detecting the tensorflow installation, or adding a build flag.  In the meantime, you can simply remove that requirement and it should install fine.

**Comment by kumarsameer** [comment]
Thanks. Have already done that.

**Comment by jan-xyz** [comment]
What about
```
try:
   import tensorflow
except ModuleNotFoundError:
   requirements.append('tensorflow>=1.0.0')
```

after the requirements block? I can create a pull request for that 3 liner.

**Comment by kumarsameer** [comment]
Removing the tensorflow requirement and putting the above 4 liner should fix this issue I think.

**Comment by rhsimplex** [comment]
Marking this resolved by #13
